xcache/Dockerfile

```diff
@@ -26,6 +43,9 @@ ADD xrootd/* /etc/xrootd/config.d/
 RUN mkdir -p "$XC_ROOTDIR"
 RUN chown -R xrootd:xrootd /xcache/
 
+# Remove the core file soft limit
+RUN [[ -z "$DEBUG" ]] || ulimit -S -c unlimited
```

Comment:
Does this work as intended?

IIUC, each [`RUN`](https://docs.docker.com/engine/reference/builder/#run) line runs inside its own `/bin/sh` instance which means that builtin commands like `ulimit` (or `cd`) that affect the current shell process won't persist, i think.

Also, just to note it here, the fact that this is run through `/bin/sh -c` means that `[[ ... ]]` is not suitable for `RUN` lines if the container is debian-based, though i think you are fine in this case since rhel systems have `/bin/sh -> bash`

Reply:
Carl and I discussed this offline. We believe that moving this to a file in `/etc/osg/image-config.d/` should ensure that the ulimit is passed along to the xrootd processes started by supervisor.